troupon/static/scss/partials/_sections.scss

```diff
@@ -0,0 +1,424 @@
+
+    [class *='col-'] {
+      padding-bottom: 20px;
+    }
+
+    .divider-2 {
+      margin: 5px auto 10px;
+    }
+  }
+}
+
+
+// dashboard subview:
+
+.subview-wrapper {
+  padding-left: 50px;
+
+  form > .divider {
+    margin-bottom: 30px;
+  }
+}
+
+
+// modals:
+
+.modals-wrapper {
+
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  z-index: z-index(modal);
+  overflow: scroll;
+
+  background-color: color(modal-bg);
+  opacity: 0;
+  visibility: hidden;
+  transition: opacity 0.5s, visibility 0s 0.5s;
+
+  &.open {
+    opacity: 1;
+    visibility: visible;
+    transition: opacity 0.5s;
+  }
+
+  #modals-close {
+    @include btn-icon-mixin (btn-icon, icon-size(lg), rect);
+
+    position: fixed;
+    overflow: visible;
+    top: 10px;
+    right: 20px;
+
+    .caption {
+      position: absolute;
+      top: 120%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      
+      font-weight: font-weight(bold);
+      font-size: font-size(label);
+
+      text-transform: uppercase;
+    }
+  }
+
+  .modals-container {
+    position: relative;
+    min-height: 100%;
+    perspective: 1200px;
+    overflow-x: hidden;
+    padding-x: 50px;
+  }
+
+  .modal-view {
+    position: absolute;
+    padding-top: 10%;
+    padding-bottom: 10%;
+    min-width: 100%;
+
+    opacity: 0;
+    visibility: hidden;
+    transform: translateY(-25%) rotateX(35deg);
+    
+    transition: transform 0.5s, opacity 0.5s;
+
+    &.open {
+      opacity: 1;
+      visibility: visible;
+      transform: rotateX(0deg);
+    }
+
+    &.close {
+      opacity: 0;
+      transform: translateY(25%) rotateX(-35deg);
+    }
+  }
+}
+.modals-parent {
+  &.open{
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+    filter: blur(20px);
+    transition: all 0.5s;
+  }
+}
+
+
+// specific modals:
+
+%modal-auth {
+  
+  .brand {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+
+  header {
+    p > a {
+      @extend .label-link;
+      margin-left: 5px;
+    }
+
+    @media (min-width: 1200px) {
+      [class *='col-'] {
+        @include contents-v-middle(60px);
+      }
+      p {
+        width: 95%;
+        text-align: right;
+      }
+    }
+  }
+}
+#modal-login {
+  @extend %modal-auth;
+  .brand-col {
+    min-height: 390px;
+  }
+}
+#modal-register {
+  @extend %modal-auth;
+  .brand-col {
+    min-height: 550px;
+  }
+}
+#modal-forgot-password {
+  @extend %modal-auth;
+  .brand-col {
+    min-height: 300px;
+  }
+}
+
+
+// modal lisings:
+
+.modal-listing {
+  .listing {
+    margin-top: 30px;
+  }
+}
+
+// messages
+.strip {
+  padding: 10px 5px 10px;
+}
+.subject {
+  font-size: 22px;
+  padding-bottom: 5px;
+}
+#compose-form {
+  padding-bottom: 8px;
+}
+.message {
+  background: #ffffff;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  margin-top:5px;
+  margin-bottom: 5px;
+}
+.message-type {
+  font-size: 14px;
+}
+.message-type:after{
+  content:' —';

```

Comment:
Colon after property should be followed by one space<br>Prefer double-quoted strings